pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -1390,6 +1390,8 @@ spec:
               type: ""
           containers:
           - name: sjr
+            securityContext:
```

Comment:
why it needs privileged access?

Reply:
@utkarshmani1997 can you launch it with serviceAccount like replica pods.

Reply:
@vishnuitta @kmova there is no service account for the replica pods. It runs as privileged pod. Also this doesn't require service account, service account is required only when any pod/deployment/sts/ds in a cluster require access to do some operation of the kubernetes resource. 
In this case we need to have PSP which will provide access to the configured resources. So this needs to be introduced in replica as well.

Reply:
Ref: https://kubernetes.io/docs/concepts/policy/pod-security-policy/#policy-reference
Please have a look at allowHostPath section

Reply:
or scc needs to be created to allowHostPath in openshift clusters
Ref: https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html#use-the-hostpath-volume-plugin
cc: @nsathyaseelan

Reply:
understood. thanks @utkarshmani1997